modules/dependents_benefits/lib/dependents_benefits/claim_processor.rb

```diff
@@ -91,12 +91,17 @@ def handle_enqueue_failure(error)
                                     { parent_claim_id:, error: error.message })
 
       parent_claim_group = SavedClaimGroup.find_by!(parent_claim_id:, saved_claim_id: parent_claim_id)
-      parent_claim_group.update!(status: 'failure')
+      parent_claim_group.update!(status: SavedClaimGroup::FAILURE)
     rescue => e
       monitor.track_processor_error('Failed to update ClaimGroup status', 'status_update',
                                     { parent_claim_id:, error: e.message, original_error: error.message })
     end
 
+    def record_enqueue_completion(claim_id)
+      claim_group = SavedClaimGroup.find_by(parent_claim_id:, saved_claim_id: claim_id)
```

Comment:
I think if you pulled in my refactors (https://github.com/department-of-veterans-affairs/vets-api/pull/24410) you could use scopes here. Or create a new one to encompass this since it's used in the `handle-enqueue_failure` as well